integration-commons/src/test/java/org/symphonyoss/integration/exception/authentication/ConnectivityExceptionTest.java

```diff
@@ -0,0 +1,68 @@
+package org.symphonyoss.integration.exception.authentication;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link ConnectivityException}.
+ * Created by crepache on 23/06/17.
+ */
+public class ConnectivityExceptionTest {
+
+  private String serviceName = "service name";
+
+  private String solution = "solution";
+
+  private Throwable cause = new Throwable("cause");
+
+  @Test
+  public void testConnectivityException() {
+    ConnectivityException exception = new ConnectivityException(serviceName);
+    String resultMessage = exception.getMessage();
+    String expectedMessage = "\n" +
```

Comment:
You could create a private method to build these String messages and avoid repeated code.